frontend/src/components/dashboard/contactsHomePage/ContactList.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { objectOf, arrayOf, string } from "prop-types";
+import { Link } from "react-router-dom";
+import { Table } from "antd";
+import * as router from "@/constants/routes";
+import * as Strings from "@/constants/strings";
+import NullScreen from "@/components/common/NullScreen";
+import CustomPropTypes from "@/customPropTypes";
+
+/**
+ * @class ContactList - paginated list of contacts
+ */
+
+const propTypes = {
+  parties: objectOf(CustomPropTypes.party).isRequired,
+  partyIds: arrayOf(string).isRequired,
+};
+
+const columns = [
+  {
+    title: "Contact Name",
+    dataIndex: "name",
+    render: (text, record) => (
+      <Link to={router.PARTY_PROFILE.dynamicRoute(record.key)}>{text}</Link>
+    ),
+  },
+  {
+    title: "Role",
+    dataIndex: "role",
+    render: (text) => <div title="role">{text}</div>,
+  },
+  {
+    title: "Email",
+    dataIndex: "email",
+    render: (text) => <div title="email">{text}</div>,
+  },
+  {
+    title: "Phone",
+    dataIndex: "phone",
+    render: (text) => <div title="phone">{text}</div>,
+  },
+];
+
+const transformRowData = (parties, partyIds) =>
+  partyIds.map((id) => ({
+    key: id,
+    emptyField: Strings.EMPTY_FIELD,
+    name: parties[id].name ? parties[id].name : Strings.EMPTY_FIELD,
+    email: parties[id].email === "Unknown" ? Strings.EMPTY_FIELD : parties[id].email,
```

Comment:
Is the API returning values as the string "Unknown"?

There's a db constraint that `party.email` cannot be `null`. So, I'm just wondering where this value would be coming from

Reply:
yeah it's coming back as 'Unknown', the phone number is also returned as "Unknown"

Reply:
Okay. I just took a look at the codebase, and that's what we're putting in during ETL when the email isn't available in MMS. I feel like that should just be a nullable field and should use `null` instead if we can't actually guarantee to always have a useful value there.

I'll start a chat on Slack.

Reply:
good idea! The Unkown has caused some issues with our null checks.